docs/connector-development/tutorials/build-a-connector-the-hard-way.md

```diff
@@ -992,61 +1034,111 @@ If the Airbyte server isn't already running, start it by running **from the Airb
 docker-compose up
 ```
 
-Then visiting [locahost:8000](http://localhost:8000) in your browser once Airbyte has started up \(it can take 10-20 seconds for the server to start\).
+Note: if you are on an M1 Mac, you should run Airbyte in `dev` mode:
+```bash
+VERSION=dev docker-compose up
```

Comment:
they also need to build airbyte locally, `SUB_BUILD=PLATFORM dev docker-compose up`

Reply:
did you mean `SUB_BUILD=PLATFORM ./gradlew build` ?

Reply:
done